dhcpv4/options.go

```diff
@@ -64,20 +75,27 @@ func OptionsFromBytes(data []byte) ([]Option, error) {
 			return nil, err
 		}
 		options = append(options, *opt)
+
+		// Options with zero length have no length byte, so here we handle the
+		// ones with nonzero length
 		if len(opt.Data) > 0 {
-			// options with zero length have no length byte, so here we handle the ones with
-			// nonzero length
 			idx++
 		}
 		idx += len(opt.Data)
 	}
 	return options, nil
 }
 
-func OptionsToBytes(options []Option) []byte {
-	// Convert a list of options to a wire-format representation. This will
-	// include the Magic Cookie
+// OptionsToBytesWithMagicCookie converts a list of options to a wire-format
+// representation with the DHCP magic cookie prepended.
+func OptionsToBytesWithMagicCookie(options []Option) []byte {
```

Comment:
I'd rather rename this function back to `OptionsToBytes`, since it's more general and going to be used more frequently, and the one below to become `OptionsToBytesWithoutMagicCookie`